utils/build/libs.mk

```
@@ -0,0 +1,26 @@
+
+# libaries for generating TARGET
+EIGEN     := $(LIBS_DIR)/Eigen
+RAPIDJSON := $(LIBS_DIR)/rapidjson
+GITHOOKS  := .git/hooks
+
+DEPENDENCIES  := $(EIGEN) $(RAPIDJSON) $(GITHOOKS)
+
+# Re-install eigen library if the tar file changes
+# EIGEN is a header only lib, no compilation needed
+$(EIGEN): $(EIGEN).tar.gz
+	$(Echo) Unpacking Eigen library $@
+	$(Verb) tar -zxvf $@.tar.gz -C lib > /dev/null
```

Same with the other, would it be better to replace this with a variable?
```suggestion
	$(Verb) tar -zxvf $@.tar.gz -C $(LIBS_DIR) > /dev/null
```

---

@mkristien so should this be changed or not? Is there a reason to have it different than below?